### PR TITLE
docs: update composition-api.md

### PR DESCRIPTION
**Pull request by wxsms**
## Description of Problem

typescript/composition-api.md

### CODE REVIEW

**Review by watonyweng — APPROVED** [review]

File: src/guide/typescript/composition-api.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # TypeScript 与组合式 API {#typescript-with-composition-api}
 
-> 这一章假设你已经阅读过了这篇 [搭配 TypeScript 使用 Vue](./overview) 的文档。
+> 这一章假设你已经阅读了位于[搭配 TypeScript 使用 Vue](./overview) 的概览。
```
Comment (Jinjiang) [inline src/guide/typescript/composition-api.md:3]:
```suggestion
> 这一章假设你已经阅读了[搭配 TypeScript 使用 Vue](./overview) 的概览。
```

File: src/guide/typescript/composition-api.md [hunk 2]
```diff
@@ -52,14 +52,14 @@ const props = defineProps<Props>()
...
 
 - 一个类型字面量
-- 对 **同一个文件** 中的一个接口或对象类型字面量的引用
+- 对**同一个文件**中的一个接口或对象类型字面量的引用
 
 这个同一个文件的限制在未来的版本中可以被移除。
 :::
 
-### props 默认值 <sup class="vt-badge warning">experimental</sup> {#props-default-values}
+### prop 默认值 <sup class="vt-badge warning">experimental</sup> {#props-default-values}
```
Comment (Jinjiang) [inline src/guide/typescript/composition-api.md:60]:
```suggestion
### Prop 默认值 <sup class="vt-badge warning">experimental</sup> {#props-default-values}
```

File: src/guide/typescript/composition-api.md [hunk 3]
```diff
@@ -74,11 +74,11 @@ const { foo, bar = 100 } = defineProps<Props>()
 </script>
```
Comment (Jinjiang) [inline src/guide/typescript/composition-api.md:74]: L71：结构 -> 解构

File: src/guide/typescript/composition-api.md [hunk 4]
```diff
@@ -224,7 +224,7 @@ const double = computed<number>(() => {
 ```vue
 <script setup lang="ts">
 function handleChange(event) {
-  // `event` 隐式为 `any` 类型
+  // `event` 隐式地拥有 `any` 类型
```
Comment (Jinjiang) [inline src/guide/typescript/composition-api.md:227]:
```suggestion
  // `event` 隐式地标注为 `any` 类型
```

**Review by Jinjiang — APPROVED** [review]

**Pull request merged**